libs/sql-schema-describer/src/mysql.rs

```diff
@@ -10,7 +10,17 @@ pub struct SqlSchemaDescriber {
 
 impl super::SqlSchemaDescriberBackend for SqlSchemaDescriber {
     fn list_databases(&self) -> SqlSchemaDescriberResult<Vec<String>> {
-        Ok(vec![])
+        let databases = self.get_databases();
+        Result::Ok(databases)
+    }
+
+    fn get_metadata(&self, schema: &str) -> SqlSchemaDescriberResult<SQLMetadata> {
+        let count = self.get_table_names(&schema).len();
+        let size = self.get_size(&schema);
+        Result::Ok(SQLMetadata {
```

Comment:
No need to change it, but I'm curious to know where the `Result` prefix comes from.